craterpy/exceptions.py

```diff
@@ -0,0 +1,17 @@
+"""Custom exceptions used by this project."""
+
+
+class CraterAttributeError(AttributeError):
+    pass
+
+
+class CraterValueError(ValueError):
+    pass
+
+
+class LatLongOutOfBoundsError(ValueError):
+    pass
+
+
+class DataImportError(Exception):
```

Comment:
I chose to subclass Exception rather than ImportError because the latter should only be used for python `import` errors.

Reply:
This will be a useful Exception to explicitly catch import issues